conans/client/rest/rest_client_common.py

```diff
@@ -242,12 +242,19 @@ def search(self, pattern=None, ignorecase=True):
         """
         url = self.router.search(pattern, ignorecase)
         response = self.get_json(url)["results"]
+        result = []
         try:
-            return [ConanFileReference.loads(reference) for reference in response]
-        except TypeError as te:
+            for reference in response:
+                name, version, user, channel, revision = get_reference_fields(reference)
+                # In Conan 2.0 it is possible to have user and not channel, skip it
+                if user and not channel:
```

Comment:
is it possible to have channel and not user?

Reply:
no